src/components/Nav/Menu/index.tsx

```diff
@@ -0,0 +1,107 @@
+import { AnimatePresence, motion } from "framer-motion"
+import { Box, type BoxProps, Flex, Text } from "@chakra-ui/react"
+import * as NavigationMenu from "@radix-ui/react-navigation-menu"
+
+import { Button } from "@/components/Buttons"
+
+import { SECTION_LABELS } from "@/lib/constants"
+
+import type { NavSections } from "../types"
+
+import SubMenu from "./SubMenu"
+import { useNavMenu } from "./useNavMenu"
+
+type NavMenuProps = BoxProps & {
+  sections: NavSections
+}
+
+const Menu = ({ sections, ...props }: NavMenuProps) => {
+  const {
+    activeSection,
+    containerVariants,
+    direction,
+    handleSectionChange,
+    isOpen,
+    menuColors,
+    onClose,
+  } = useNavMenu(sections)
+
+  return (
+    <Box {...props}>
+      <NavigationMenu.Root
+        dir={direction}
+        orientation="horizontal"
+        delayDuration={500}
+        onValueChange={handleSectionChange}
+      >
+        <NavigationMenu.List asChild>
+          <Flex as="ul" listStyleType="none">
+            {SECTION_LABELS.map((sectionKey) => {
+              const { label, items } = sections[sectionKey]
+              const isActive = activeSection === sectionKey
+              return (
+                <NavigationMenu.Item key={sectionKey} value={label}>
```

Comment:
Out of curiosity: could you explain the purpose of the `value`? it is controlling what?

Reply:
I don't think we actually need that here actually... it gets assigned as part of the element ID, so it can be selected easier, but also can be assigned as "defaultValue" for the Root or Sub, so it knows which one should be open at first... we are not assigning this though, and the reason I had assigned a value here is no longer needed. Going to remove.